src/components/CategoriesTable/CategoriesTableRow.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactElement } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen, faTrashCan } from '@fortawesome/free-solid-svg-icons';
+import cn from 'classnames';
+import { useCategoryFormContext } from 'components/CategoryForm/CategoryFormContext';
+import { CategoryDto } from 'dtos/CategoryDto';
+
+interface Props {
+  category: CategoryDto;
+  padding: string;
+}
+
+export const CategoriesTableRow = ({ category, padding }: Props): ReactElement => {
+  //const { removecategory } = useProducts();
+  const formContext = useCategoryFormContext();
+
+  const handleDelete = (): void => {
+    console.log('Retiring of categories not yet implemented');
+    /*
+    removeProduct.mutate(product.id, {
```

Comment:
Yeah remove all comments. This is great, but we once the api has adjusted we can create tickets for others to implement this.